tests/letstest/scripts/version.py

```diff
@@ -14,6 +13,8 @@ def certbot_version(build_script_dir):
     """Return the version number stamped in certbot/__init__.py."""
     return re.search('''^__version__ = ['"](.+)['"].*''',
                      file_contents(join(dirname(build_script_dir),
```

Comment:
I just have one nitpick which we can leave if you really want since things are working.

The name of the parameter this function takes is `build_script_dir` which I believe refers to the directory that contained `build.py` where this code was originally used. From that directory, `dirname` was used to access the parent directory.

I don't really care if we use `..` or `dirname` (although maybe it'd be nice to be consistent), but do you think it's worth changing the name of this parameter since I don't think it makes sense in the new context?

Reply:
Good call on dirname, didn't even notice that. And yeah tbh I don't think it really matters, but that cruft could be confusing and this is a cruft-removal PR, so I'll change it.